execution_workflows/Roar/workflow/Snakefile

```diff
@@ -0,0 +1,124 @@
+"""
+Snakemake pipeline for APA-Scan.
+For help see: https://snakemake.readthedocs.io/en/stable/index.html.
+"""
+
+
+# ------------------------------------------------------------------------------
+# Packages, Libraries and Sources
+
+import pandas as pd
+
+
+
+
+
+# ------------------------------------------------------------------------------
+# Config/Metadata
+
+# Read in config file
+configfile: "config/config.Roar.yaml"
+
+# Convert config to dataframe
+samples = pd.read_csv(os.path.abspath(config["samples"]),sep=",")
+
+
+
+
+
+#-------------------------------------------------------------------------------
+# Parameters, filenames & sample data
+
+#### NOTES #####################################################################
+
+# MAY NEED TO ADJUST THE INDEXING DEPENDING ON THE LEVELS, BUT CAN PROBABLY MAKE
+# THIS CUSTOMISABLE FROM THE COMMAND LINE OR A CONFIG FILE ETC. THIS IS BECAUSE
+# THE LAYOUT OF THESE WILL DICTATE THE ORIENTATION OF LEVELS THAT ARE COMPARED
+# BY ROAR. IE FOR THE TEST DATA, THIS MAKES SO SRSF3 is CONDITION1 AND
+# THE CONTROL IS CONDITION 2
```

Comment:
Moving these notes to the README makes it more readable (you mention to reformat rule declarations for better readability, not sure if this is included).